src/test/javascript/spec/component/text-submission-assessment/text-feedback-conflicts.component.spec.ts

```diff
@@ -225,7 +192,7 @@ describe('TextFeedbackConflictsComponent', () => {
 
     it('should use jhi-text-feedback-conflicts-header', () => {
         const headerComponent = fixture.debugElement.query(By.directive(TextFeedbackConflictsHeaderComponent));
-        expect(headerComponent).toBeTruthy();
+        expect(headerComponent).not.toBe(undefined);
```

Comment:
query returns null on error. This test cannot fail at the moment.
```suggestion
        expect(headerComponent).not.toBe(null);
```
When I comment out the entire template of the tested component, only three tests fail, and that's because of errors, not test failures. This test, for example, still passes as if everything works correctly